src/gen/oracle.rs

```diff
@@ -62,47 +61,56 @@ impl DartCodeOracle {
         Self::sanitize_identifier(&nm.to_lower_camel_case())
     }
 
+    /// Get the idiomatic Dart rendering of an FFI callback function name
+    fn ffi_callback_name(nm: &str) -> String {
+        format!("Pointer<NativeFunction<Uniffi{}>>", nm.to_upper_camel_case())
+    }
+
     /// Get the idiomatic Dart rendering of an exception name
-    // TODO: Refactor to be more idomatic to the way dart handles errors
     pub fn error_name(nm: &str) -> String {
-        // errors are a class in Dart.
         let name = Self::class_name(nm);
         match name.strip_suffix("Error") {
             None => name,
             Some(stripped) => format!("{stripped}Exception"),
         }
     }
 
+    pub fn find_lib_instance() -> dart::Tokens {
+        quote!(_UniffiLib.instance)
+    }
+
```

Comment:
as we hard-code this in the builder anyways, why make it a function on the Oracle and not just use `_UniffiLib.instance` everywhere instead?